### ISSUE TITLE
Abnormal amyloid beta 40 isoform level 

### ISSUE THREAD

**Issue by sarahstofel** [obophenotype/human-phenotype-ontology#7070]
**Preferred term label:**
Abnormal amyloid beta 40 isoform level

**Synonyms**
Abnormal beta amyloid 40 level, abnormal Abeta 40 level

**Definition (free text, please give PubMed ID)**
Any anomaly in the amount of the amyloid beta isoform that is 40 amino acids in length.
PMID:29420472

**Parent term (use hpo.jax.org/app)**
Abnormal amyloid beta protein level

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM: 104300

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0003-2307-1226

**Comment by pnrobinson** [comment]
@sarahstofel  -- does this term also include the 39 isoform?

**Comment by pnrobinson** [comment]
added
